plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-additional-information-block/attributes.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import formStepAttributes from '../../form-step/attributes';
+
+export default {
+	...formStepAttributes( {
+		defaultTitle: __( 'Additional order information', 'woocommerce' ),
+		defaultDescription: __(
+			'Enter any additional information about your order.',
```

Comment:
I feel like this is a strange description which may only apply if certain fields are added. For example, if you only added a checkbox, what are you entering?

Should the default perhaps be empty?

Reply:
Good idea, I also hid this from view if no additional fields are present.